src/common.rs

```diff
@@ -65,11 +69,14 @@ fn into_rust_with_args(field_type: syn::Ty, arguments: quote::Tokens) -> quote::
   let field_type_ident = get_cdrs_type_ident(field_type.clone());
   match field_type_ident.as_ref() {
     "Blob" | "String" | "bool" | "i64" | "i32" | "i16" | "i8" | "f64" | "f32" | "Decimal"
-    | "IpAddr" | "Uuid" | "Timespec" | "PrimitiveDateTime" | "NaiveDateTime" | "DateTime" => {
+    | "IpAddr" | "Timespec" | "PrimitiveDateTime" | "NaiveDateTime" | "DateTime" => {
       quote! {
         #field_type_ident::from_cdrs_r(#arguments)?
       }
     }
+    "Uuid" => quote! {
```

Comment:
Was there a problem with Uuid?

Reply:
It will expand to this:

Uuid::from_cdrs_r(#arguments)?

That fails if:
- Uuid is not in scope
- A different Uuid is in scope than uuid::Uuid. 
 
I added the whole qualified path, this will always work. The other types other than the primitives have the same problem, but I didn't run into problems so I kept them how they are.